api/internal/localizer/localizer_test.go

```diff
@@ -267,31 +281,98 @@ patches:
     allowNameChange: true
   path: patch.yaml
 `,
-		"patch.yaml": `apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Deployment
-metadata:
-  name: not-used
-spec:
-  template:
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:1.21.0
+		"patch.yaml": podConfiguration,
+	}
+	checkLocalizeInTargetSuccess(t, kustAndPatch)
+}
+
+func TestLocalizeOpenAPI(t *testing.T) {
+	type testCase struct {
+		name  string
+		files map[string]string
+	}
+	for _, test := range []testCase{
+		{
+			name: "no_path",
+			files: map[string]string{
+				"kustomization.yaml": `apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+openapi:
+  version: v1.20.4
 `,
+			},
+		},
+		{
+			name: "path",
```

Comment:
Can we also add a test here with a remote path, since `openapi` supports remote paths in `kustomize build`?

Reply:
Done.